Codec/Archive/Tar/Write.hs

```diff
@@ -98,6 +100,11 @@ putHeaderNoChkSum Entry {
     , fill        12 $ '\NUL'
     ]
   where
+    numField :: (Integral a, Bits a, Show a) => FieldWidth -> a -> String
+    numField = case format of
+      V7Format -> putOct
```

Comment:
What happens here btw if we're out of range with V7 format? Shouldn't we fail? Will we write a borked archive?

Reply:
It will do exactly what the old code did, which I assume is to produce a borked archive.